core/station/api/spec.did

```diff
@@ -1600,6 +1653,7 @@ type Resource = variant {
   Account : AccountResourceAction;
   AddressBook : ResourceAction;
   ChangeCanister : ChangeCanisterResourceAction;
+  ManageCanister : ManagedCanisterResourceAction;
```

Comment:
Should this be like this instead?

```suggestion
  ManagedCanister : ManagedCanisterResourceAction;
```

Reply:
We have ChangeCanister as another variant so we're using verbs as the variant: the variant is "manage a canister" and the specified is an "action" for this "managed canister".

Reply:
i see your point, i was viewing the pattern more like an association, so, if not specific you would just have `ResourceAction`, but if specific you would use the same name as in the variant with the value but would suffix that name with `ResourceAction`. 

So in that case it would be `ManagedCanister` as the resource, and the actions that you can perform on that would be `ManagedCanisterResourceAction`.

The convention would be:

```
type Resource = variant {
  Example: ExampleResourceAction,
}
```

Reply:
Makes sense! Fixed.